src/content/en/2020/capabilities.md

```diff
@@ -0,0 +1,336 @@
+
+{{ figure_markup(
+  image="badging_api.png",
+  alt="Percentage of page loads in Chrome using Badging API",
+  caption='Percentage of page loads in Chrome using Badging API. (Source: <a href="https://chromestatus.com/metrics/feature/timeline/popularity/2726">Chrome Platform Status metrics for Badge Set</a>, <a href="https://chromestatus.com/metrics/feature/timeline/popularity/2727">Chrome Platform Status metrics for Badge Clear</a>)',
+  description="Chart of Badging API usage, based on the percentage of page loads in Chrome using this feature. It compares the set and clear methods. The usage of both methods is growing over time, with the set method generally being called more often. In October 2020, there's a sudden growth for both methods, peaking at 0.025 percent of page loads for the set method and 0.016 percent for clear.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=1145004925&format=interactive",
+  sheets_gid="1154751352"
+  )
+}}
+
+In April 2020, Google Chrome 81 shipped the new Badging API, followed by Microsoft Edge 84 in July. After Chrome shipped the API, the usage numbers shot up. In October 2020, on 0.025 percent of all page loads in Google Chrome, the `setAppBadge()` method is called. The `clearAppBadge()` method is less often called, during around 0.016 percent of page loads.
+
+### Notification Triggers API
+
+The Push API requires the user to be online to receive a notification. Some applications, such as games, reminder or to-do apps, calendars, or alarm clocks, could also determine the target date for a notification locally and schedule it. To support this feature, the Chrome team experiments with a new API called [Notification Triggers](https://web.dev/notification-triggers/) ([Explainer](https://github.com/beverloo/notification-triggers/blob/master/README.md), not on a standards track yet). This API adds a new property called `showTrigger` to the `options` map that can be passed to the `showNotification()` method on the Service Worker's registration. The API is designed to allow for different kinds of triggers in the future, albeit for now, only time-based triggers are implemented. For scheduling a notification based on a certain date and time, developers can create a new instance of a `TimestampTrigger` and pass the target timestamp to it:
+
+```js
```

Comment:
(Side note: @barrypollard it'd be nice to support syntax highlighting)

Reply:
We do support a limited form and use it in the methodology chapter but it basically needs to be lovely written by hand at the moment:

https://github.com/HTTPArchive/almanac.httparchive.org/blob/a5cbe77943ab7202c9ed55fdef71b20560a62033/src/templates/en/2020/methodology.html#L88-L104

That may be sufficient for now but a nicer code formatter would be nicer alright.